libvmaf/src/predict.c

```diff
@@ -61,30 +61,143 @@ static int denormalize(const VmafModel *model, double *prediction)
     return 0;
 }
 
+int find_linear_function_parameters(Point p1, Point p2, double *alpha, double *beta) {
```

Comment:
Use `static` for functions that don't see use outside of this file.

Reply:
It was used outside in test_predict.c.